workspaces/adventure-pack/goodies/typescript/Number.isIntegerOrIntegerObject/index.test.ts

```diff
@@ -27,7 +27,7 @@ describe("Number.isIntegerOrIntegerObject", () => {
     expect(Number.isIntegerOrIntegerObject(value)).toBe(true);
   });
 
-  it.each([INTEGERS])("returns true for integer object %p", (value) => {
+  it.each(INTEGERS)("returns true for integer object %p", (value) => {
```

Comment:
The test passed previously because converting an array to a number gives 0, which is an integer.